hedera-node/configuration/testnet/upgrade/feeSchedules.json

```diff
@@ -409,6 +409,51 @@
                 }
             } ]
         }
+    }, {
+        "transactionFeeSchedule": {
+            "hederaFunctionality": "TokenAirdrop",
+            "fees": [
+                {
+                    "subType": "DEFAULT",
+                    "nodedata": {
+                        "constant": 7574478,
```

Comment:
This should be using `TOKEN_FUNGIBLE_COMMON` or `TOKEN_NON_FUNGIBLE_UNIQUE` sections of `CryptoTransfer` since the `DEFAULT` is `0.0001`

Reply:
Similarly in all places inside `feeSchedule.json`

Reply:
@Neeharika-Sompalli fixed 👍